packages/react/src/functions/string-to-function.ts

```diff
@@ -195,6 +201,9 @@ const makeFn = (code: string, useReturn: boolean) => {
   }
   return new Proxy({}, {
       get(target, key) {
+          if (key === 'copySync') {
+            return () => obj.copySync();
+          }
```

Comment:
where is that `copySync` key coming from exactly? I assume I might need the same workaround in the new SDKs when I add isolate vm

Reply:
it's coming from the `ivm.Reference` https://github.com/laverdet/isolated-vm#referencecopysync , I refactored a bit to improve readability and added more comments